Common/Tests/Extensions/StringTests.swift

```diff
@@ -0,0 +1,151 @@
+//                     ("Jordan Currency", "1000000", "JOD", "د.أ.‏ 1,000,000.000"),
+//                     ("Kazakhstan Currency", "1000000", "KZT", "₸1,000,000.00"),
+//                     ("Kenya Currency", "1000000", "KES", "Ksh 1,000,000.00"),
+//                     ("Laos Currency", "1000000", "LAK", "₭1,000,000"),
+//                     ("Malaysia Currency", "1000000", "MYR", "RM 1,000,000.00"),
+//                     ("Mexico Currency", "1000000", "MXN", "$1,000,000.00"),
+//                     ("Morocco Currency", "1000000", "MAD", "MAD 1,000,000.00"),
+//                     ("Israel Currency", "1000000", "ILS", "₪1,000,000.00"),
+//                     ("Taiwan Currency", "1000000", "TWD", "$1,000,000.00"),
+//                     ("Turkey Currency", "1000000", "TRY", "₺1,000,000.00"),
+//                     ("New Zealand Currency", "1000000", "NZD", "$1,000,000.00"),
+//                     ("Nigeria Currency", "1000000", "NGN", "₦1,000,000.00"),
+//                     ("Norway Currency", "1000000", "NOK", "kr 1,000,000.00"),
+//                     ("Pakistan Currency", "1000000", "PKR", "Rs 1,000,000"),
+//                     ("Peru Currency", "1000000", "PEN", "S/ 1,000,000.00"),
+//                     ("Philippines Currency", "1000000", "PHP", "₱1,000,000.00"),
+//                     ("Poland Currency", "1000000", "PLN", "zł 1,000,000.00"),
+//                     ("Isle of Man Currency", "1000000", "GBP", "£1,000,000.00"),
+//                     ("Romania Currency", "1000000", "RON", "RON 1,000,000.00"),
+//                     ("Russia Currency", "1000000", "RUB", "₽1,000,000.00"),
+//                     ("Serbia Currency", "1000000", "RSD", "RSD 1,000,000"),
+//                     ("Singapore Currency", "1000000", "SGD", "$1,000,000.00"),
+//                     ("South Africa Currency", "1000000", "ZAR", "R 1,000,000.00"),
+//                     ("South Korea Currency", "1000000", "KRW", "₩1,000,000"),
+//                     ("Sri Lanka Currency", "1000000", "LKR", "Rs. 1,000,000.00"),
+//                     ("Sweden Currency", "1000000", "SEK", "kr 1,000,000.00"),
+//                     ("Switzerland Currency", "1000000", "CHF", "CHF 1,000,000.00"),
+//                     ("Thailand Currency", "1000000", "THB", "฿1,000,000.00"),
+//                     ("Tunisia Currency", "1000000", "TND", "د.ت.‏ 1,000,000.000"),
+//                     ("United Arab Emirates Currency", "1000000", "AED", "د.إ.‏ 1,000,000.00"),
+//                     ("Uganda Currency", "1000000", "UGX", "USh 1,000,000"),
+//                     ("United States Currency", "1000000", "USD", "$1,000,000.00"),
+//                     ("Vietnam Currency", "1000000", "VND", "₫1,000,000")
+//        ]
+//        cases.forEach {
+//            XCTAssertEqual($1.formatCurrencyAmount(with: $2), $3, "\($0) test case - currency should be equal to \($3)")
+//        }
+//    }
+
+    func testFormatCurrencyAmount() {
```

Comment:
Delete commented code and update func name to `testFormatToCurrency`. This looks good so we will go with this implementation

Reply:
Okay